### PR TITLE
Subframes and quotient locales via nuclei

### PR DESCRIPTION
**Pull request by EgbertRijke**

### CODE REVIEW

File: src/order-theory/nuclei-large-locales.lagda.md [hunk 1]
```diff
@@ -37,24 +38,116 @@ module _
...
+        ( hom-large-meet-semilattice-nucleus-Large-Locale j)
+
+    preserves-top-nucleus-Large-Locale :
+      map-nucleus-Large-Locale (top-Large-Locale L) ＝ top-Large-Locale L
+    preserves-top-nucleus-Large-Locale =
+      preserves-top-hom-Large-Meet-Semilattice
+        ( hom-large-meet-semilattice-nucleus-Large-Locale j)
+```
+
+### The `j`-closed elements of a nuceus
```
Comment (fredrik-bakke) [inline src/order-theory/nuclei-large-locales.lagda.md:117]: nucleus*

File: src/order-theory/nuclei-large-locales.lagda.md [hunk 2]
```diff
@@ -37,24 +38,116 @@ module _
...
+    type-subtype (is-closed-element-nucleus-Large-Locale-Prop {l1})
+```
+
+## Properties
+
+### The `j`-closed elements of a nucleus on a Locale form a subframe, and hence a quotient locale of `L`.
+
+```agda
+
 ```
```
Comment (fredrik-bakke) [inline src/order-theory/nuclei-large-locales.lagda.md:153]: unfinished code?

**Review by fredrik-bakke — APPROVED** [review]
Looks good to me :)

**Comment by EgbertRijke** [comment]
Done!

**Pull request merged**